proposed/2022/PrivateAssetIndepdentFromExcludeAsset.md

```diff
@@ -0,0 +1,321 @@
+# Opt-in experience for PrivateAssets and ExcludeAssets options are independent
+
+- Author Name <https://github.com/erdembayar>
+- Start Date 2022-12-07
+- GitHub Issue <https://github.com/NuGet/Home/issues/6938>
+- GitHub PR <https://github.com/NuGet/NuGet.Client/pull/4976>
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+Currently asset consumption via `PrivateAssets` option experience for assets from transitive package in a project/package is not fully controllable by the project author for parent consuming project, here `PrivateAssets` option calculation is not independent from `IncludeAssets/ExcludeAssets` option, if an asset is not consumed in current project then it doesn't flow up the regardless of `PrivateAssets` option.
+This proposal introduces new a boolean `ExcludedAssetsFlow` metadata to improve experience for `PrivateAssets` option to make it independent from `IncludeAssets/ExcludeAssets` option, it'll make assets not specified in `PrivateAssets` will flow regardless of `IncludeAssets/ExcludeAssets` option.
+
+Below means assets `contentFiles, build, buildMultitargeting, buildTransitive, analyzers, native` excluding `runtime and compile` assets would flow to referencing parent project even though they're not consumed by current project.
+
+`<PackageReference Include="PackageA" PrivateAssets="Runtime;Compile" ExcludedAssetsFlow="true" IncludeAssets = "none"/>`
+
+If `ExcludedAssetsFlow` if not present, then it default false and it's ignored by older tooling. Also, if it has non-boolean value then it would error out.
```

Comment:
nit: Just say the default value is false or somewhere and this sentence can be removed completely.